File: 01_strings_and_arrays/04_most_freq_char/most_freq_char.js
const mostFrequentChar = (s) => {
    const count = {}
    let highestChar = ''
    let charCount = 0
    
    for(let char of s) {
      if(!(char in count)) {
        count[char] = 1
      } else {
        count[char] += 1
      }
    }
    
    for(let char in count) {
      if(count[char] > charCount) {
        charCount = count[char]
        highestChar = char
      }
    }
    
    return highestChar
  };
  
  module.exports = {
    mostFrequentChar,
  };
  

File: 01_strings_and_arrays/04_most_freq_char/most_freq_char.py
def most_frequent_char(s):
    count = {}
    highest_char = ''
    char_count = 0

    for char in s:
        if char not in count:
            count[char] = 1
        else:
            count[char] += 1
        
    for char in count:
        if count[char] > char_count:
            char_count = count[char]
            highest_char = char

    return highest_char


print(most_frequent_char('bookeeper')) # -> 'e'